PINCache/PINCache.m

```diff
@@ -294,6 +310,14 @@ - (NSUInteger)diskByteCount
     return byteCount;
 }
 
+- (BOOL)containsObjectForKey:(NSString *)key
+{
+    if (!key)
+        return NO;
```

Comment:
Will change in line 79-81 and other places that I added braces to be consistent across the project.